src/Symfony/Component/Config/ConfigCacheFactoryInterface.php

```diff
@@ -28,5 +28,5 @@ interface ConfigCacheFactoryInterface
      *
      * @return ConfigCacheInterface The cache instance
      */
-    public function cache($file, $callable);
+    public function cache(string $file, $callable);
```

Comment:
missing callable type

Reply:
I followed @derrabus advice and skipped the `callable` type. See: https://github.com/symfony/symfony/pull/31984#discussion_r292192250